quiz/backend/stats/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+
+
+class TimestampModel(models.Model):
+    """Abstract class for saving timestamps.
+
+    :param created_at: Date and time when given object was created.
+    :type created_at: models.DateTimeField(auto_now_add=True)
+
+    :param updated_at: Date and time of when given object was updated.
+    :type updated_at: models.DateTimeField(auto_now_add=True)
+
+    """
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
+
+class Badge(TimestampModel):
+    """Represents badges that players/users can earn during games.
+    Inherits from TimestampModel and :class:`stats.models.TimestampModel`
+
+    Following fields are inherited:
+        * created_at
+        * updated_at
+
+    :param name: Name of badge
+    :type name: models.CharField(max_length=40, unique=True)
+
+    :param weight: Base for calculating bonus points after player/user receives given badge.
```

Comment:
On the weight thingy I'm not really sure about the use case for that, some kind of leaderboard, but that should be based on ELO not badges.

Reply:
I was following the old model since I can't find new ER Diagram for Quiz